paper.tex

```diff
@@ -2414,5 +2474,47 @@ \section{Testing Survey Results}
 \label{tab:testingSurveyResults}
 \end{table}
 
+\section{NOxAR Assignment}
+\label{sec:NOxAR}
+
+On Monday, September 9th, Atticus approached us about an additional project he 
+wanted us to tackle. He wanted us to adapt Kyoto VR's NOxAR app to use ground
+plane detection. Currently, the NOxAR app is an AR app for Microsoft Hololens
+that uses image targets with Vuforia. When viewing the correct image target,
+the app projects a 3D model of a Noh Theater\footnote{A form of Japanese 
+theater originating in the 14th century structured around song and slow 
+dancing~\autocite{japanguide2018}.} performer dancing to background 
+music~\autocite{noxar2018}. What Atticus wanted us to do was have the app use 
+ground plane detection in order to display the Noh performer so that it could 
+be displayed anywhere. Additionally, he wanted us to scale up the model so that
+when displayed in AR, the performer would appear life-sized, as if she were 
```

Comment:
get rid of everything after the comma on this line; this is implicit at this point since we explained how AR works